extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/OpenshiftProcessor.java

```diff
@@ -146,24 +146,14 @@ public void openshiftRequirementsJvm(ContainerImageOpenshiftConfig openshiftConf
         boolean hasCustomJvmArguments = config.jvmArguments.isPresent();
 
         builderImageProducer.produce(new BaseImageInfoBuildItem(baseJvmImage));
-        Optional<OpenshiftBaseJavaImage> baseImage = OpenshiftBaseJavaImage.findMatching(baseJvmImage);
```

Comment:
I'm surprised the tests are passing as I had test failures when I removed the images but good if they pass.

Reply:
Yes so I can confirm the tests are not passing and will require some adjustments:

In `integration-tests/kubernetes/quarkus-standard-way`, you will have this failure:

```
[ERROR] Failures: 
[ERROR]   OpenshiftWithS2iTest.assertGeneratedResources:74 
[List check single element] (1 failure)
-- failure 1 --
[List check single element] (1 failure)
-- failure 1 --
Expecting any element of:
  [EnvVar(name=KUBERNETES_NAMESPACE, value=null, valueFrom=EnvVarSource(configMapKeyRef=null, fieldRef=ObjectFieldSelector(apiVersion=null, fieldPath=metadata.namespace, additionalProperties={}), resourceFieldRef=null, secretKeyRef=null, additionalProperties={}), additionalProperties={}),
    EnvVar(name=JAVA_APP_JAR, value=/deployments/quarkus-run.jar, valueFrom=null, additionalProperties={})]
to satisfy the given assertions requirements but none did:
```

Reply:
@famod this is interesting as for GIB is concerned. GIB didn't run the `integration-tests/kubernetes/quarkus-standard-way` module because there is no explicit dependency in the POM. We cannot really add the dependencies to the POM because they are actually conditional and used specifically in some of the tests. I'm not completely sure how we could create a dependency there?

Reply:
@gsmet I''ll have a look when I'm back from Switzerland, not before sunday.